cpp/ps0007/problem_s_8.cpp

```diff
@@ -39,17 +42,22 @@ int main() {
         paramCount = 1;
     } else {
         cout << "Shape not recognized." << endl;
-        return 0;
+        return 1;
     }
```

Comment:
- [x] checked the user's input for choosing the shape
- [x] used suitable size of dimensions arrays with the shape